src/DynamoApplications/StartupUtils.cs

```diff
@@ -224,6 +224,11 @@ private static IUpdateManager InitializeUpdateManager()
         /// <returns></returns>
         public static DynamoModel MakeCLIModel(string asmPath, string userDataFolder, string commonDataFolder, HostAnalyticsInfo info = new HostAnalyticsInfo())
         {
+            if (string.IsNullOrEmpty(info.HostName))
+            {
+                info.HostName = "Dynamo";
+            }
```

Comment:
Is there a reason why you need to make sure there is a valid string in `info.HostName` at this point.
Dynamo already sets a default value here https://github.com/DynamoDS/Dynamo/blob/master/src/DynamoCore/Logging/DynamoAnalyticsClient.cs#L141

Reply:
Or maybe we can move the default value in the actual struct definition...or some other "common" location